internal/provider/config_resource_test.go

```diff
@@ -57,6 +57,14 @@ func TestAccConfigWithoutBasicAuthResource(t *testing.T) {
 					resource.TestCheckResourceAttr("altitude_mte_config.tester", "environment_id", TEST_ENVIRONMENT_ID),
 				),
 			},
+			{
+				Config: testAccKVResourceConfigCacheMaxAge(TEST_ENVIRONMENT_ID, SECONDARY_HOST),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					resource.TestCheckResourceAttr("altitude_mte_config.tester", "config.routes.0.host", SECONDARY_HOST),
+					resource.TestCheckResourceAttr("altitude_mte_config.tester", "environment_id", TEST_ENVIRONMENT_ID),
+					resource.TestCheckResourceAttr("altitude_mte_config.tester", "config.routes.0.cache_max_age", "360"),
+				),
+			},
```

Comment:
It would probably be better to create a new test with it's own steps for the static maxage.

The function could also take the static maxage as a parameter so that the test could look like:

1. Create resource with no static max age
2. Update resource to include one
3. Delete resource

Reply:
Have split it out into a separate test, it creates a resource without the max age, then updates the same resource (i believe) to include one.
I believe the delete happens automatically at the end of the tests?